features/exploredash/src/main/java/org/dash/wallet/features/exploredash/utils/DashDirectConfig.kt

```diff
@@ -32,6 +32,7 @@ import javax.inject.Singleton
 class DashDirectConfig @Inject constructor(private val context: Context) {
     companion object {
         val PREFS_KEY_LAST_DASH_DIRECT_ACCESS_TOKEN = stringPreferencesKey("last_dash_direct_access_token")
+        val PREFS_KEY_DASH_DIRECT_EMAIL = stringPreferencesKey("dash_direct_email")
```

Comment:
@hadia this will be stored as plain text in the settings. Personal user info needs to be stored securely. For example:
https://developer.android.com/topic/security/data
https://proandroiddev.com/securing-androids-datastore-ad56958ca6ee

Reply:
nice catch ,I forget about that part

Reply:
create EncryptionProvider